src/main/java/frc/robot/OI.java

```diff
@@ -65,12 +72,18 @@ public static JoystickButton getOrientationButton(){
         public static JoystickButton getZeroButton(){
             return new JoystickButton(kJoystick, kZeroButtonID);
         }
+
         public static JoystickButton getIntakeButton(){
             return new JoystickButton(kJoystick, kIntakeButtonID);
         }
+
         public static JoystickButton getOuttakeButton(){
             return new JoystickButton(kJoystick, kOuttakeButtonID);
         }
+        
+        public static JoystickButton getParkButton(){
+            return new JoystickButton(kJoystick, kParkButtonID);
+
```

Comment:
```suggestion
        }
```
This is what's causing build to fail